voter-dapp/src/App.js

```diff
@@ -26,16 +40,16 @@ function App() {
   if (drizzle) {
     return (
       <drizzleReactHooks.DrizzleProvider drizzle={drizzle}>
-        <MuiThemeProvider theme={theme}>
+        <ThemeProvider theme={theme}>
```

Comment:
Just for my information - what's the difference between MuiThemeProvider and ThemeProvider?

Reply:
Hahahaha it was super annoying because there's a `@material-ui/core/styles` and a `@material-ui/styles`, and it took a while to figure out how to transmit the theme down.

IIUC `ThemeProdiver` makes the theme available to the `makeStyles` function (which doesn't currently but could depend on the theme).